style.css

```diff
@@ -0,0 +1,83 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Inter", sans-serif;
+}
+
+nav {
```

Comment:
[Optional] You have used nav as the container for the toolbar section, but you did not apply any margins as indicated by the requirements. It makes your UI a little bit different from the [Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468) design. 
![image](https://user-images.githubusercontent.com/62849872/143327951-3e6dce71-7727-4d20-93b8-6e6c0e496f92.png)
Your design 
![image](https://user-images.githubusercontent.com/62849872/143328009-3afefded-1527-479c-8696-310777c5edf0.png)
Figma design 
![image](https://user-images.githubusercontent.com/62849872/143328074-b1426993-4fa0-4d65-abe9-312616d3532f.png)
Always make an effort to make your UI look the same like the Figma design because the same principles will apply at your job.